backend/python/app/graphql/mutations/story_mutation.py

```diff
@@ -40,6 +41,7 @@ class Arguments:
 
     story = graphene.Field(lambda: CreateStoryTranslationResponseDTO)
 
+    @require_authorization_by_role_gql({"User", "Admin"})
```

Comment:
Not sure if `CreateStoryTranslation` mutation can be admin only.

Reply:
user calls this mutation on the homepage when deciding to translate a story to a language, so no need to make this admin only